src/components/primitives/Theme.tsx

```diff
@@ -181,6 +181,12 @@ const styles = css`
   [data-p='4'] {
     padding: var(--studio-spacing-4);
   }
+
+  .lucide {
```

Comment:
@going-confetti I moved the global style to here so that it applies equally to k6 Studio and extension.